src/kakarot/accounts/library.cairo

```diff
@@ -15,64 +18,311 @@ from starkware.starknet.common.syscalls import (
     storage_read,
     storage_write,
     StorageReadRequest,
+    CallContract,
+    get_tx_info,
+    get_contract_address,
+    get_caller_address,
+    replace_class,
 )
 from starkware.cairo.common.memset import memset
+from utils.eth_transaction import EthTransaction
 
 from kakarot.interfaces.interfaces import IERC20, IKakarot
+from kakarot.errors import Errors
 
+// @dev: should always be zero for EOAs
 @storage_var
-func bytecode_len_() -> (res: felt) {
+func Account_bytecode_len() -> (res: felt) {
 }
 
 @storage_var
-func storage_(key: Uint256) -> (value: Uint256) {
+func Account_storage(key: Uint256) -> (value: Uint256) {
 }
 
 @storage_var
-func is_initialized_() -> (res: felt) {
+func Account_is_initialized() -> (res: felt) {
 }
 
 @storage_var
-func evm_address() -> (evm_address: felt) {
+func Account_evm_address() -> (evm_address: felt) {
```

Comment:
wondering if we should give a meaningful name to the output, or just res. And if a name, then replace `res: felt` I guess